GPU-SUPPORT.md

```diff
@@ -0,0 +1,19 @@
+# GPU Support in Livepeer
+
+Livepeer enables node operators to transcode video on GPUs while concurrently mining cryptocurrencies. As there is a very wide range of GPU hardware out there, this document aims to crowdsource a list of specific GPU models that have been tested and verified to work on Livepeer. If you've tested an additional model, please submit an update to this document.
+
+**Overview**
+
+* Livepeer supports transcoding on NVidia GPUs with NVENC/NVDEC. Any GPU [listed here](https://developer.nvidia.com/video-encode-and-decode-gpu-support-matrix-new) with those chips should theoretically work. Note that different models have different driver versions and may be subject to session limits.
```

Comment:
> Note that different models have different driver versions and may be subject to session limits.

AFAIK the models themselves aren't associated with a specific driver version, but the driver installed on a machine is what imposes session limits based on the model used.

Suggestion:

"Note that different models may be subject to different session limits that restrict the amount of video that can be transcoded"